src/appComponents/twitterwall/singletweet.js

```diff
@@ -68,8 +60,19 @@ function tweet($timeout,$rootScope) {
                 callchangetweet();
             }
             }
-        function callchangetweet(){
-              $timeout(changeTweet,8000);
-        }   
+       
+        function getObject(){
+                scope.$watch('array',function(){
+                if(scope.array){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'id_str' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2414597727/issues/source?bid=3233857&fileBranchId=3235518#l67)